### ISSUE TITLE
[BUG] composer install fails due to zendframework/zend-barcode version mismatch

### ISSUE THREAD

**Issue by nesh-younify** [tig-nl/postnl-magento2#122]
**To Reproduce**
1. composer install the fresh Magento 2.3.2
2. Run `composer require tig/postnl-magento2`

**Expected result**
Installation finished without errors.

**Actual result**
Error due to version mismatch:

```
Using version ^1.8 for tig/postnl-magento2
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.
 Problem 1
   - Installation request for tig/postnl-magento2 ^1.8 -> satisfiable by tig/postnl-magento2[1.8.0].
   - Conclusion: remove zendframework/zend-barcode 2.7.1
   - Conclusion: don't install zendframework/zend-barcode 2.7.1
   - tig/postnl-magento2 1.8.0 requires zendframework/zend-barcode 2.5.2|2.7 -> satisfiable by zendframework/zend-barcode[2.5.2, 2.7.0].
   - Can only install one of: zendframework/zend-barcode[2.7.0, 2.7.1].
   - Can only install one of: zendframework/zend-barcode[2.5.2, 2.7.1].
   - Installation request for zendframework/zend-barcode (locked at 2.7.1) -> satisfiable by zendframework/zend-barcode[2.7.1].
Installation failed, reverting ./composer.json to its original content.
```

The error seems to be related to the `vendor/temando/module-shipping-m2/composer.json` having a require `"zendframework/zend-barcode": "^2.7.0",` which pulls newer (`2.7.1`) version when installed.

**Workaround**
Add `"zendframework/zend-barcode": "2.7"` to the top-level Magento `composer.json` or, maybe, add this extension before initial Magento composer install.

**Fix**
If possible, your composer.json should define something like `"zendframework/zend-barcode": "2.5.2|^2.7"` instead having fixed 2.7 version.

- Magento version: Magento 2.3.2 CE
- PHP version: 7.2
- TIG PostNL version: 1.8 (composer pull)

**Comment by tig-daanvandenbergh** [comment]
@nesh-younify,

Thanks for notifying us!

In the next release it will defined as `~2.5.2|~2.7.0`.

**Comment by SolsWebdesign** [comment]
is there another way around? like composer require with something like ignore or dev? I can't install now (Magento 2.3.2 CE).

**Comment by nesh-younify** [comment]
> is there another way around? like composer require with something like ignore or dev? I can't install now (Magento 2.3.2 CE).

You can add "zendframework/zend-barcode": "2.7" to the top-level Magento composer.json and composer will then install correct version and then require for this ext should work.

**Comment by tig-daanvandenbergh** [comment]
Thanks for the suggestion of the work-around @nesh-younify.

@SolsWebdesign a fix for this is scheduled for the next release, which should follow very soon.

**Comment by tig-daanvandenbergh** [comment]
@SolsWebdesign @nesh-younify,

This issue is resolved in version 1.8.1.
